package.json

```diff
@@ -22,13 +22,17 @@
     "@creit.tech/stellar-wallets-kit": "^1.2.5",
     "@ledgerhq/hw-app-str": "^7.0.4",
     "@ledgerhq/hw-transport-webusb": "^6.29.4",
+    "@rjsf/core": "^5.24.3",
```

Comment:
included for `feature-invoke-host-function` branch so that we can use it as an example and test our implementation